lib/cyclone-core/src/resolver_function.rs

```diff
@@ -62,6 +62,10 @@ impl CycloneRequestable for ResolverFunctionRequest {
         &self.execution_id
     }
 
+    fn kind(&self) -> &str {
+        "resolverfunction"
```

Comment:
One day, one day, one day we'll rename this

Reply:
Actually, I'll just do it here and now since I'm already touching both sides of the relationship anyways.